Tools/docker_run.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
+
+PWD=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+SRC_DIR=$PWD/../
+
+docker run -it --rm -w ${SRC_DIR} \
+	-v ${SRC_DIR}:${SRC_DIR}:rw \
+	-v /tmp/.X11-unix:/tmp/.X11-unix:ro \
+	-v ${HOME}/.ccache:${HOME}/.ccache:rw \
+	-e CCACHE_DIR=${HOME}/.ccache \
```

Comment:
@dagar this reminds me, I have trouble with the current containers compiling because the default cache directory is owned by root.